bls/src/main/java/tech/pegasys/teku/bls/BLS.java

```diff
@@ -94,6 +100,9 @@ public static BLSSignature aggregate(final Stream<BLSSignature> signatures) {
    */
   public static boolean aggregateVerify(
       List<BLSPublicKey> publicKeys, List<Bytes> messages, BLSSignature signature) {
+    if (publicKeys.isEmpty()) {
```

Comment:
nit:
```suggestion
    if (publicKeys.isEmpty() || messages.isEmpty()) {
```

Reply:
Yeah, this check is redundant here as `publicKeys` must be the same length as `messages`. But you've made me realise that for consistency with the standard we should also do the length check here (it's currently only checked elsewhere), and the uniqueness check for messages. So I'm going to rework some of this.